spring-cloud-task-docs/src/main/asciidoc/features.adoc

```diff
@@ -212,3 +212,39 @@ marking the final state of the task.
 }
 ```
 
+=== Launching a task from a Spring cloud stream
+
+Allows a user to launch tasks from a stream.  This is done by creating a sink that
+listens for a message that contains a `TaskLaunchRequest` as a part of its payload.
```

Comment:
It will throw an SpelEvaluationException because it does not handle other message types.  I 'll update the docs.